client/my-sites/post-share/index.jsx

```diff
@@ -0,0 +1,436 @@
+			<Connection
+				connection={ connection }
+				onToggle= { this.toggleConnection }
+				isActive={ this.isConnectionActive( connection ) }
+				key={ connection.keyring_connection_ID }
+			/>
+		);
+	}
+
+	setMessage = message => this.setState( { message } );
+
+	dismiss = () => {
+		this.props.dismissShareConfirmation( this.props.siteId, this.props.post.ID );
+	};
+
+	sharePost = () => {
+		this.props.sharePost( this.props.siteId, this.props.post.ID, this.state.skipped, this.state.message );
+	};
+
+	isButtonDisabled() {
+		if ( this.props.requesting ) {
+			return true;
+		}
+
+		return this.props.connections.filter( this.isConnectionActive ).length < 1;
+	}
+
+	previewSharingPost = () => {
+	}
+
+	renderMessage() {
+		if ( ! this.hasConnections() ) {
+			return;
+		}
+
+		const targeted = this.hasConnections() ? this.props.connections.filter( this.isConnectionActive ) : [];
+		const requireCount = includes( map( targeted, 'service' ), 'twitter' );
+		const acceptableLength = ( requireCount ) ? 140 - 23 - 23 : null;
+
+		return (
+			<PublicizeMessage
+				message={ this.state.message }
+				preview={ this.props.post.title }
+				requireCount={ requireCount }
+				onChange={ this.setMessage }
+				acceptableLength={ acceptableLength } />
+		);
+	}
+
+	renderShareButton() {
+		const { translate } = this.props;
+		return (
+			<div className="post-share__button-actions">
+				<Button onClick={ this.previewSharingPost }>
+					{ translate( 'Preview' ) }
+				</Button>
+
+				<ButtonGroup className="post-share__share-combo">
+					<Button
+						className="post-share__button"
+						primary
+						onClick={ this.sharePost }
+						disabled={ this.isButtonDisabled() }
+					>
+						{ translate( 'Share post' ) }
+					</Button>
+
+					<AsyncLoad
+						require="blocks/scheduler-popover"
+						site={ this.props.site }
+						type="button" // ButtonGroup hopes that this element should be a button.
+					>
+						<Button
+							primary
+							className="post-share__schedule-button"
+							title={ translate( 'Set date and time' ) }
+							tabIndex={ 3 }
+						>
+							<Gridicon icon="calendar" />
+						</Button>
+					</AsyncLoad>
+				</ButtonGroup>
+			</div>
+		);
+	}
+
+	renderUpgradeNudge() {
+		if ( this.isSchedulingEnabled ) {
+			return null;
+		}
+
+		const {
+			businessDiscountedRawPrice,
+			businessRawPrice,
+			translate,
+			userCurrency,
+		} = this.props;
+
+		return (
+			<Banner
+				className="post-share__footer-banner"
+				callToAction={
+					translate( 'Upgrade for %s', {
+						args: formatCurrency( businessDiscountedRawPrice || businessRawPrice, userCurrency ),
+						comment: '%s will be replaced by a formatted price, i.e $9.99'
+					} )
+				}
+				description={ translate( 'Live chat support and no advertising.' ) }
+				list={ [
+					translate( 'Live chat support' ),
+					translate( 'No Advertising' )
+				] }
+				plan={ PLAN_BUSINESS }
+				title={ translate( 'Upgrade to a Business Plan!' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'To upload themes, upgrade to Business Plan' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).